index.bs

```diff
@@ -120,6 +120,7 @@ spec: HTML; urlPrefix: https://html.spec.whatwg.org/
     text: set up a window environment settings object; url: set-up-a-window-environment-settings-object
     text: set up a worker environment settings object; url: set-up-a-worker-environment-settings-object
     text: set up a worklet environment settings object; url: set-up-a-worklet-environment-settings-object
+    text: window open steps; url: multipage/window-object.html#window-open-steps
```

Comment:
None of the other definitions use `multipage`. Why is that needed here?

Reply:
All the terms that we don't need to add custom link text for use the multipage URLs, and in context of most links this looked inconsistent. But I'll send a PR to fix this whole block instead.

Reply:
https://github.com/w3c/webdriver-bidi/pull/152

Reply:
That's now merged and I've merged to resolve the resulting conflicts.